hilo.rb

```diff
@@ -1,34 +1,79 @@
 #!/usr/bin/env ruby
-puts "Welcome to the Hi-Lo game.\nGuess a random whole number from 1 to 100 (inclusive)."
-puts "You will have a total of 6 guesses."
-puts ""
 
-random_num = rand(1..100)
+# Game to guess random number
+class HiLoGame    
+    @@already_guessed = Array.new
```

Comment:
Using `@@` variables is really not a good idea. That makes it a class level variable which means that you're changing the guesses for all games. Like if you're running two games simlitanously one games guesses will also apply to the second game.

A better alternative is to define a instance variable in the initializer. 
```
def initialize(max_guesses)
  @max_guesses = max_guesses
  @already_guessed = already_guessed
end
```